components/brave_wallet_ui/components/extension/confirm-transaction-panel/style.ts

```diff
@@ -319,3 +321,43 @@ export const AssetIcon = AssetIconFactory<AssetIconProps>({
   width: '40px',
   height: 'auto'
 })
+
+// [FIXME]: Duplicated from components/brave_wallet_ui/components/extension/sign-panel/style.ts
```

Comment:
maybe re-export `components/brave_wallet_ui/components/extension/sign-panel/style.ts` instead of redefining?

Reply:
Yes, we should share the styles for this one. Will do a follow-up on Nightly.